index.html

```diff
@@ -59,7 +59,151 @@ <h1>Hey there. I'm Osugo. <br> <span>I'm a software developer</span></h1>
 
 
     </div>
-    </div>
-</body>
 
-</html>
\ No newline at end of file
+
+    <section class="recent-works">
+
+
+        <div class="topper">
+            <h2>My Recent Works</h2>
+        </div>
+
+        <div class="indicator"></div>
+
+        <div class="recent-works-cards">
+            <div class="works work1">
+                <div class="work-details">
+                    <h1>Multi-Post Stories Gain+Glory</h3>
```

Comment:
Please consider removing this typing error. You have a start tag of `h1` and an end tag of `h3`. Consider using some hx tag other than h1 here. Or just replace it with the following line:-

## Solution:-

```
<h3>Multi-Post Stories Gain+Glory</h3>
```

Reply:
@ABDULALI3468 I missed that ouch....Can't believe it has costed me a whole review.But thanks for the keen eye